project/src/Game/World/Characters/enemies/Enemy.gd

```diff
@@ -105,16 +113,25 @@ func get_expected_facint_based_on_target_map_position(target_map_pos:Vector2) ->
 
 
 func get_path_to_closest_tree_world_pos() -> Array:
```

Comment:
Myślę, że można spokojnie tutaj powydzielić wszystko na wiele mniejszych metod. To przy okazji nazwy metod dokumentują co dana mała część kodu, ma robić. Zyskujesz komentarze wbudowane w kod :)

Np. to całe można przenieść do osobnej metody, wtedy nie będzie pustej deklaracji `trees`. Dodatkowo na pierwszy rzut oka trudno jest zrozumieć co tu się konkretnie dzieje.
```
if tile_map.get_cellv(targetted_tree) != tile_map.TREE_ID:
		trees = tile_map.get_used_cells_by_id_in_map_range(tile_map.TREE_ID)
	else: 
		trees = [targetted_tree]
```